tuf/api/metadata.py

```diff
@@ -397,7 +397,9 @@ def verify_delegate(
         delegated_metadata: "Metadata",
         signed_serializer: Optional[SignedSerializer] = None,
     ) -> None:
-        """Verify that ``delegated_metadata`` is signed with the required
+        """Verify that ``delegated_metadata`` is signed with the required threshold.
+
+        Verify that ``delegated_metadata`` is signed with the required
```

Comment:
I'm not super fond of verbatim duplicates. Vertical space in source code files is valuable.  :) Maybe we can get a bit creative in cases like these.

Reply:
Sounds very reasonable. I'll try to push creativity. 👍🏻

But please, as I'm not that familiar with python-tuf codebase, it will require more in-depth review not to adjust semantics.